app/ecolabel-translator/src/App.js

```diff
@@ -9,14 +9,33 @@ function App() {
     { sender: 'Assistant', text: 'Welcome to the chat!' }
   ]);
   const [isChatActive, setIsChatActive] = useState(false);
+  const [token, setToken] = useState(1);
 
-  const handleLabelSubmit = (event) => {
+  const handleLabelSubmit = async (event) => {
     event.preventDefault();
-    setResults({
-      description: 'This is a sample eco-label description.',
-      epaRecommendation: 'EPA recommends this label for sustainable products.'
-    });
+  
+    try {
+      const response = await fetch(`http://127.0.0.1:5000/search?q=${label}`, {
```

Comment:
Can we get this into a variable somewhere? Then it can be easily changed when we move to prod